app.py

```diff
@@ -86,7 +86,7 @@ class Meta:
 # Endpoints
 @app.route('/', methods=['GET'])
 def home():
-    return "<h1>TravelRx</h1><p>This site is the homepage for the back end of TravelRx.  Please visit our search endpoint at /api/v1/search or a user's medicine cabinet at /api/v1/medicines.</p>"
+    return "<h1>TravelRx</h1><p>This site is the homepage for the back end of TravelRx.  Please visit our search endpoint at /api/v1/search?drug=<drug_name or a user's medicine cabinet at /api/v1/medicines.</p>"
```

Comment:
I believe the endpoint is missing a >
it should be /api/v1/search?drug=<drug_name>

Reply:
Thanks for catching that! Fixed!